docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 worker: &worker
   build: .
-  command: ./manage.py celeryd
+  command: -- celery worker --app=olympia
```

Comment:
The `--` makes the args go to the `celery` command instead of the calling function. Without it you end up with `su: unrecognized option '--app=olympia'`.